### DISCUSSION TITLE
Cycle GAN vs Contrastive Unpaired Translation (CUT)

### DISCUSSION THREAD

**Discussion by Dongbin-Lee-git** [bcaitech1/p4-fr-soccer#24] [Experiment]
![image](https://user-images.githubusercontent.com/57521132/119910863-0fbbdd80-bf93-11eb-8a15-10ed3f70dcaf.png)
---
#### Train
- size : 768, 768
- dataset : 45000 set
- batch size : 1
- iteration : 40200 vs 33100
- time : 16 hour vs 9 hour
---
#### Test 
- CycleGAN (left) vs CUT (right)
---
![image](https://user-images.githubusercontent.com/57521132/119911320-13039900-bf94-11eb-912e-7ed06ebccf86.png)
![image](https://user-images.githubusercontent.com/57521132/119911468-4d6d3600-bf94-11eb-8f74-abf84d312046.png)
![image](https://user-images.githubusercontent.com/57521132/119911597-8b6a5a00-bf94-11eb-9f5b-6b6cdc6c0da2.png)

**Comment by bellamy-k** [comment]
고생하셨습니다!
